### ISSUE TITLE
Could we have an `OPAMJOBS` env var

### ISSUE THREAD

**Issue by dbuenzli** [ocaml/opam#2961]
that has the value of the `jobs` variable in `build:` instructions ?

**Comment by dra27** [comment]
`OPAMJOBS` is already a variable, of course - perhaps we could consider setting all the `OPAM*` variables to their effective values during instructions?

**Comment by dbuenzli** [comment]
> all the OPAM* variables to their effective values during instructions?

Not sure it makes sense, but I'd like `OPAMJOBS`. OTOH now that I think of it it's questionable whether build systems should use `jobs`/`OPAMJOBS` which I guess are actually variables that drive the opam parallel engine.

**Comment by dra27** [comment]
Indeed, it does drive the opam parallel engine - this quickly heads towards jobservers!
